ios/PurchasesHybridCommon/PurchasesHybridCommon/StoreProduct+HybridAdditions.swift

```diff
@@ -21,8 +21,8 @@ import StoreKit
             "introPrice": NSNull(),
             "price": self.price,
             "priceString": self.localizedPriceString,
-            "productCategory": self.productCategoryString,
-            "productType": self.productTypeString,
+            "productType": self.productCategoryString,
+            "productSubtype": self.productTypeString,
```

Comment:
Yeah, I'm not sure where we draw the line here, TBH 😓 Because these names _do_ match up with StoreKit but don't match up with Billing Client.

So its either...
1) Do we want all the SDKs to have similar naming?
2) Do we want the native SDKs to match the native API and app store / console naming? And then have hybrids have hybrid naming?

But this feels like a bigger discussion that should be had about future of all SDKs 🤷‍♂️

Reply:
Yeah somewhere in this conversion there's going to be some weirdness because we need to unify iOS and Android. I think PHC is the best place to do that.